src/object/srv_obj_migrate.c

```diff
@@ -187,8 +187,11 @@ obj_tree_create(daos_handle_t toh, void *key, size_t key_size,
 
 	*rootp = tmp_root;
 out:
-	if (rc < 0)
-		dbtree_delete(toh, BTR_PROBE_EQ, &key_iov, NULL);
+	if (rc < 0) {
+		rc2 = dbtree_delete(toh, BTR_PROBE_EQ, &key_iov, NULL);
+		if (rc2)
+			D_WARN("failed to delete tree: "DF_RC"\n", DP_RC(rc2));
```

Comment:
debtree_delete only removes the key, not the whole tree as implied by the warning.  Might also be useful to print the DF_KEY.

Reply:
Thank you! Going to update it soon.